src/main/resources/au/com/rayh/XCodeBuilder/config.jelly

```diff
@@ -22,211 +22,218 @@
-		        <f:checkbox title="Yes" />
-		    </f:entry>
-
-		    <f:entry title="${%Allow failing build results?}" field="allowFailingBuildResults"
-		      description="Checking this option will prevent this build step from failing if xcodebuild exits with a non-zero return code.">
-		        <f:checkbox title="Yes" />
-		    </f:entry>
-		    <f:entry title="${%Generate Archive?}" field="generateArchive"
-		    description="Checking this option will generate an xcarchive of the specified scheme. A workspace and scheme are are also needed for archives">
-		    <f:checkbox title="Yes" />
-		    </f:entry>
-
-		    <f:entry title="${%Configuration}" field="configuration"
-		      description="This is the name of the configuration as defined in the Xcode project.">
-		        <f:textbox default="Release"/>
-		    </f:entry>
+        <f:advanced title="Settings">
+            <f:entry title="${%Clean before build?}" field="cleanBeforeBuild" description="This will delete the build directories before invoking the build.">
+                <f:checkbox title="Yes" />
+            </f:entry>
+
+            <f:entry title="${%Allow failing build results?}" field="allowFailingBuildResults"
+                description="Checking this option will prevent this build step from failing if xcodebuild exits with a non-zero return code.">
+                <f:checkbox title="Yes" />
+            </f:entry>
+            <f:entry title="${%Generate Archive?}" field="generateArchive"
+                description="Checking this option will generate an xcarchive of the specified scheme. A workspace and scheme are are also needed for archives">
+                <f:checkbox title="Yes" />
+            </f:entry>
+
+            <f:entry title="${%Configuration}" field="configuration" description="This is the name of the configuration as defined in the Xcode project.">
+                <f:textbox default="Release" />
+            </f:entry>
 
             <f:entry>
-                <f:optionalBlock inline="true" checked="${instance.buildIpa == true}" name="buildIpa" title="${%Pack application and build .ipa?}" description="Checking this option will create a .ipa for each .app found in the build directory.">
-                    <f:entry title="${%.ipa filename pattern}" field="ipaName" description="A pattern for the ipa file name. You may use $${VERSION} and $${BUILD_DATE} (yyyy.MM.dd) in this string">
+                <f:optionalBlock inline="true" checked="${instance.buildIpa == true}" name="buildIpa" title="${%Pack application, build and sign .ipa?}"
+                    description="Checking this option will create a .ipa for each .app found in the build directory.">
+                    <f:entry title="${%Export method}" field="ipaExportMethod"
+                        description="The export method of the .app to generate the .ipa file. Should be one in 'development', 'ad-hoc', 'enterprise' or 'app-store'.">
+                        <f:textbox default="enterprise" />
```

Comment:
I think defaulting to enterprise is going to just hassle everyone. Shouldn't ad-hoc be the default?